.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,34 @@
+(Thanks for sending a pull request! Please make sure you read the contribution
+guidelines [CONTRIBUTING.rst](../CONTRIBUTING.rst), then fill out a PR like the
+template below)
+
+**IMPORTANT:** before pushing the PR you have to run tox for py27, py36 and pep8
+and get no errors!
+
+## What does this implement/fix? Explain your changes
+
+...
+
+## Does this close any currently open issues?
+
+Closes #
+
+## More comments
+
+...
+
+**Notes on Subject:**
+
+* succinct description of the change - no more than 60 chars
+
+* use the imperative, present tense: "change" not "changed" nor "changes" nor
+  "changing"
+
+* don't capitalize first letter
+
+* no dot (.) at the end
+
+**Notes on body:**
+
+* do not save words in the body. The more information the better to understand
```

Comment:
This language is a little hard for me to parse. Perhaps "* be descriptive about why this change is useful and how it is an improvement -- the more information the better"

Reply:
Totally agree :) Thanks @seandst !